tests/testthat/test_euclidean_space.R

```diff
@@ -0,0 +1,8 @@
+# unit test of euclidean_space.R
+
+es=EuclideanSpace(dimension = 3)
```

Comment:
No abbreviation, it hinders readability. `euclidean.space` to use R style convention: https://google.github.io/styleguide/Rguide.xml#identifiers